source/Octopus.Tentacle/Communications/TentacleCommunicationsModule.cs

```diff
@@ -22,11 +23,17 @@ protected override void Load(ContainerBuilder builder)
             {
                 var configuration = c.Resolve<ITentacleConfiguration>();
                 var services = c.Resolve<IServiceFactory>();
-                var halibutRuntime = new HalibutRuntimeBuilder()
+                var halibutRuntimeBuilder = new HalibutRuntimeBuilder()
                     .WithServiceFactory(services)
                     .WithServerCertificate(configuration.TentacleCertificate)
-                    .WithMessageSerializer(serializerBuilder => serializerBuilder.WithLegacyContractSupport())
-                    .Build();
+                    .WithMessageSerializer(serializerBuilder => serializerBuilder.WithLegacyContractSupport());
+
+                //if we have the environment variable to enable async halibut, enable it :)
+                if (Environment.GetEnvironmentVariable(EnvironmentVariables.TentacleEnableAsyncHalibut) == bool.TrueString)
```

Comment:
Maybe TryParse the bool instead of comparing to the string, as `"true" == bool.TrueString` is false

Reply:
Nit: my personal preference, if it works for you, it would be awesome to have brackets around the new line of the 'if' (old habits 😁 ). But I know this is mainly a Steve preference...

Reply:
hehe, I don't mind changing that. I have no strong opinion either way :)

Reply:
Fixed in 2b4689b6